script.js

```diff
@@ -59,32 +97,67 @@ ui.buttonPrevious.addEventListener('click', () => {
 		// -1 is used because ui.carouselImageContainer starts at 0 not 1
 		addActiveImageClass(ui.carouselImageContainer[lastCarouselImage - 1])
 
-		currentImage = lastCarouselImage
+		ui.currentImage = lastCarouselImage
 
-		addActiveNavButtonClass(currentImage)
-		currentImage--
+		addActiveNavButtonClass(ui.currentImage)
+		ui.currentImage--
 	} else {
 		// -1 is used because ui.carouselImageContainer starts at 0 not 1
-		addActiveImageClass(ui.carouselImageContainer[currentImage - 1])
+		addActiveImageClass(ui.carouselImageContainer[ui.currentImage - 1])
 
-		addActiveNavButtonClass(currentImage)
-		currentImage--
+		addActiveNavButtonClass(ui.currentImage)
+		ui.currentImage--
 	}
 })
 
 ui.buttonNext.addEventListener('click', () => {
 	removeActiveImageClass()
-	const isLast = currentImage + 1 > ui.carouselImageContainer.length
+	const isLast = ui.currentImage + 1 > ui.carouselImageContainer.length
 
 	if (isLast) {
 		addActiveImageClass(ui.carouselImageContainer[0])
 
-		currentImage = 1
-		addActiveNavButtonClass(currentImage)
+		ui.currentImage = 1
+		addActiveNavButtonClass(ui.currentImage)
 	} else {
-		addActiveImageClass(ui.carouselImageContainer[currentImage])
+		addActiveImageClass(ui.carouselImageContainer[ui.currentImage])
 
-		currentImage++
-		addActiveNavButtonClass(currentImage)
+		ui.currentImage++
+		addActiveNavButtonClass(ui.currentImage)
 	}
 })
+
+function listenForItemDelete() {
+	Array.from(ui.deleteButtons).forEach((button) => {
+		button.addEventListener('click', (event) => {
+			const itemToDeleteId = event.target.dataset.id
+
+			if (window.confirm('Delete item?')) {
+				deleteData('shop', itemToDeleteId)
+				deleteShopItemFromUi(itemToDeleteId)
+			}
+		})
+	})
+}
+
+function deleteShopItemFromUi(itemToDeleteId) {
+	let listItems = Array.from(ui.shopItemsList.getElementsByTagName('li'))
+	listItems.map((item, index) => {
```

Comment:
Consider using `map` when it is actually needed as this ask for no more than `forEach`.  
But anyways, I'd say that just selecting the element you want to id and calling `remove()` on it would be a more clean approach. 🤔